Public/Src/Engine/Processes/SandboxedProcessPipExecutionResult.cs

```diff
@@ -309,7 +311,8 @@ internal static SandboxedProcessPipExecutionResult MismatchedMessageCountFailure
             IReadOnlyList<ReportedFileAccess> allReportedFileAccesses,
             IReadOnlyList<ProcessDetouringStatusData> detouringStatuses,
             long maxDetoursHeapSize,
-            ContainerConfiguration containerConfiguration)
+            ContainerConfiguration containerConfiguration,
+            string stdLog="")

```

Comment:
>stdLog="" [](start = 19, length = 9)

Perhaps `string stdLog = null`. Then, in the assignment below you have `StdLog = stdLog ?? string.Empty;`.